storage_service/locations/models/package.py

```diff
@@ -845,6 +878,8 @@ def check_fixity(self, force_local=False, delete_after=True):
             failures = []
             message = ""
         except bagit.BagValidationError as failure:
+            LOGGER.info('bagit.BagValidationError on %s:\n%s', path, failure.message)
+            LOGGER.debug(subprocess.check_output(['tree', '-a', '--du', path]))
```

Comment:
I like the idea of checking the output worked, but if the subprocess fails it will raise an exception and halt the fixity check for a debug message, which seems problematic. Remove this or wrap it in exception handling?